pyproject.toml

```
@@ -0,0 +1,90 @@
+[project]
+name = "zppy_interfaces"
+dynamic = ["version"]
+authors = [
+    { name="Ryan Forsyth", email="forsyth2@llnl.gov" }
+]
+description = "A package for providing extra functionality on top of external packages"
+license = {file = "LICENSE"}
+readme = "README.md"
+requires-python = ">=3.9"
+classifiers = [
+    # these are only for searching/browsing projects on PyPI
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+    "Programming Language :: Python :: 3.12",
+    "Programming Language :: Python :: 3.13",
+
+    "License :: OSI Approved :: BSD License",
+    "Operating System :: OS Independent",
+]
+
+dependencies = [
+    "matplotlib-base >=3.8.2,<3.10",
+    "netcdf4",
+    "numpy >=2.0,<3.0",
+    "xarray >=2023.02.0",
+    "xcdat >=0.7.0,<1.0",
```

```suggestion
    "xcdat >=0.7.3,<1.0",
```

xcdat v0.7.3 is now available on `conda-forge` with the same numpy 2.0 constraint: https://anaconda.org/conda-forge/xcdat